element_deeplabcut/readers/dlc_reader.py

```diff
@@ -154,7 +154,7 @@ def do_pose_estimation(
     dlc_model,
     project_path,
     output_dir,
-    videotype=None,
+    videotype="",
```

Comment:
Also in PR #40

Reply:
I know. This is required for the SciOps ASAP.